tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -585,6 +585,14 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string ApplicationMonitoringConfigFileEnabled = "DD_APPLICATION_MONITORING_CONFIG_FILE_ENABLED";
 
+        /// <summary>
+        /// Configuration key to disable the agent discovery service.
+        /// When disabled, the tracer will not query the agent for available endpoints.
+        /// This is useful in environments where the discovery endpoint is not available (e.g., Azure Functions with Rust agent).
+        /// Default value is true (discovery service enabled).
+        /// </summary>
+        public const string DiscoveryServiceEnabled = "DD_DISCOVERY_SERVICE_ENABLED";
```

Comment:
Change the name to `DD_AGENT_FEATURE_POLLING_ENABLED `.

"Discovery service" is confusing because "service discovery" is a completely unrelated feature.

Reply:
Done